sherlock/document.py

```diff
@@ -217,6 +302,26 @@ def from_dict(cls, doc: "Document", dct: Dict[str, Any]) -> "Relation":
 
 @dataclass(frozen=True)
 class Event:
+    """
+    Representation for Events within a Document.
```

Comment:
A n-ary relation mention / event with a list of Mention arguments.

Reply:
Will take documentation from sherlock_version_2 branch